subsystems/motor/VelocitySensorMotor.java

```diff
@@ -1,48 +1,49 @@
 package org.usfirst.frc4904.standard.subsystems.motor;
 
-
 import org.usfirst.frc4904.standard.LogKitten;
 import org.usfirst.frc4904.standard.custom.motioncontrollers.MotionController;
 import org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers.IdentityModifier;
 import org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers.SpeedModifier;
-import edu.wpi.first.wpilibj.SpeedController;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.CustomSpeedController;
```

Comment:
change all instances of `CustomSpeedController` to `SpeedController`. Update naming